corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -77,15 +19,24 @@
     {% registerurl 'location_lineage' domain '-locid-' %}
     {% registerurl 'location_descendants_count' domain '-locid-'%}
 
+    {% registerurl 'edit_location' domain '-locid-' %}
+    {% registerurl 'location_search' domain %}
+    {% registerurl 'create_location' domain %}
+    {% initial_page_data 'api_root' api_root %}
+    {% initial_page_data 'locations' locations|JSON %}
+    {% initial_page_data 'can_edit_root' can_edit_root|yesno:"true,false" %}
```

Comment:
You can remove the `yesno` filters - the JSON logic in `initial_page_data` will handle converting this from a python boolean to a js boolean.